package.json

```diff
@@ -162,7 +162,7 @@
       "brotli": false,
       "path": "./packages/teams-js/dist/esm/packages/teams-js/src/index.js",
       "import": "{ app, authentication, pages }",
-      "limit": "57 KB"
+      "limit": "190 KB"
```

Comment:
Please only bump this limit by a small amount (0.5KB or less)

Reply:
Sorry it exceeded 119 bytes and I read it as 119 kb. I revert the change and updated it to be 57.15kb. Please take another look.